src/schemathesis/specs/graphql/schemas.py

```diff
@@ -59,7 +59,7 @@ def base_path(self) -> str:
         return self._get_base_path()
 
     def _get_base_path(self) -> str:
-        return cast(str, urlsplit(self.location).path)
+        return cast(str, urlsplit(self.location or "").path)
```

Comment:
This change doesn't seem relevant to the topic of this PR. Why is it changed here?

Reply:
My PRs are failing in mypy due to an optional value being passed to this function